petpal/applications/views.py

```diff
@@ -43,7 +170,38 @@ def pet_applications_list_view(request, pet_id):
 @api_view(['GET', 'PUT'])
 def application_detail_view(request, app_id):
     if request.method == 'GET':
-        return Response({'msg': 'application detail'}, status=status.HTTP_200_OK)
+        app = Application.objects.get(pk=app_id)
```

Comment:
Catch if app_id is out of range. Also only users that have access to the specific application should be able to view